system/hardware/tici/hardware.h

```diff
@@ -21,6 +21,23 @@ class HardwareTici : public HardwareNone {
   static int get_voltage() { return std::atoi(util::read_file("/sys/class/hwmon/hwmon1/in1_input").c_str()); };
   static int get_current() { return std::atoi(util::read_file("/sys/class/hwmon/hwmon1/curr1_input").c_str()); };
 
+  static std::string get_serial() {
+    static std::string serial("");
```

Comment:
why is this static?

Reply:
I thought it would be slow so this is to cache it